### ISSUE TITLE
Dreame Vacuum cleaner - set Rooms for cleanings

### ISSUE THREAD

**Issue by MauriDP** [jghaanstra/com.xiaomi-miio#259]
Hi Jelger

I installed my new Dreame L10s on Homey. It works so far…

But the function “clean a room” is not working. See attached image.

Do you have any idea where the problem is? Maybe I should put the name of the room in another way?

thx 
Mauri
![IMG_0935](https://github.com/jghaanstra/com.xiaomi-miio/assets/156342836/0706186a-0747-4b0d-afd0-8e75fcc7130a)

**Comment by jghaanstra** [comment]
Read this: https://github.com/jghaanstra/com.xiaomi-miio/wiki/Vacuum-cleaners

**Comment by MauriDP** [comment]
Thank you for the quick reply.

I have this error message:

![IMG_0930](https://github.com/jghaanstra/com.xiaomi-miio/assets/156342836/3bba5d64-f4a3-4a3f-9e6b-9cea5f693419)

**Comment by jghaanstra** [comment]
I checked the code but cant see anything wrong and I cant test it myself as I dont own the device. You can try the support topic on the support forum to get help from the community.

https://community.homey.app/t/app-pro-xiaomi-mi-home-app/118

**Comment by jghaanstra** [comment]
Nevermind, I might know what is wrong and will try to fix it with the next release.

**Comment by MauriDP** [comment]
Ok thx 👍

**Comment by MauriDP** [comment]
> Nevermind, I might know what is wrong and will try to fix it with the next release.

Great, thank you very much. I hope asap 😃😉

**Comment by jghaanstra** [comment]
A possible fix is added in release 3.2.21 which is now pending approval in the app store. Please test it.

https://homey.app/a/com.xiaomi-miio/test/

**Comment by MauriDP** [comment]
Hi JelgerGreat, I’m going to test it. 👍Thanks.Mauri Von: Jelger Haanstra ***@***.***>Gesendet: Mittwoch, Januar 17, 2024 5:14 PMAn: jghaanstra/com.xiaomi-miio ***@***.***>Cc: Mauri ***@***.***>; Author ***@***.***>Betreff: Re: [jghaanstra/com.xiaomi-miio] Dreame Vacuum cleaner - set Rooms for cleanings (Issue #259) 
A possible fix is added in release 3.2.21 which is now pending approval in the app store. Please test it.
https://homey.app/a/com.xiaomi-miio/test/

**Comment by MauriDP** [comment]
Hi Jelger

I have just tried your new release. Unfortunately still not working. ☹️ 

see attached picture.

thx for your help.
Mauri
![IMG_0955](https://github.com/jghaanstra/com.xiaomi-miio/assets/156342836/ce4bd02d-f5e7-4368-9386-bbbcc51110af)

**Comment by jghaanstra** [comment]
Then you are out of luck, I cant fix this without having the actual device.

**Comment by MauriDP** [comment]
☹️ ok, that’s a pity…

then I have to find another way.

thx anyway.

**Comment by bergh-io** [comment]
> ☹️ ok, that’s a pity…
> 
> then I have to find another way.
> 
> thx anyway.

did you find another way? I’m in the same situation

**Comment by MauriDP** [comment]
> > ☹️ ok, that’s a pity…
> > then I have to find another way.
> > thx anyway.
> 
> did you find another way? I’m in the same situation

Yes, I found another way. Not the best one probably, but it works for me. ☺️ I have an Alexa device installed on my Homey. So, I send the commands to DREAME with Virtual buttons via Alexa. Just have to write the ,spoken’ commands (like clean the bad room) in a flow. Unfortunately there are no status information about DREAME. But this is not that important to me.

**Comment by shaarkys** [comment]
@MauriDP , is this an issue ? Just in case you have this still installed, after installing latest version, send me debug log from the app after executing the flow.

**Comment by shaarkys** [comment]
Lack of answer
